src/Grid.js

```diff
@@ -165,7 +169,7 @@ Grid.prototype.redraw = function(newOptions, callback) {
   var items;
 
   window.requestAnimationFrame(function() {
-    if (this.columns !== newOptions.columns) {
+    if (this.columns !== newOptions.columns || this.columnClasses !== newOptions.columnClasses) {
```

Comment:
Yes, I did this for cases when only column classes change, not their number:

```javascript
savvior.init("#myGrid", {
  "screen and (max-width: 20em)": {
    columns: 2,
    columnClasses: 'columns columns__some-modificator'
  },
  "screen and (min-width: 20em) and (max-width: 40em)": {
    columns: 2,
    columnClasses: 'columns columns__some-other-modificator'
  },
});
```

Of course, it's not very useful config since it can be easily replaced with css:

```css
@media screen and (max-width: 20em) {
  .columns__some-modificator { /* some rules */ }
}
@media screen and (min-width: 20em) and (max-width: 40em) {
  .columns__some-modificator { /* some other rules */ }
}
```

But such configs are possible and I think user's wtf/minute rate will increase if column classes don't change from `columns columns__some-modificator` to `columns columns__some-other-modificator`.